src/main/java/drawingbot/files/exporters/Graphics2DExporter.java

```diff
@@ -18,7 +18,7 @@
 public class Graphics2DExporter {
 
     public static void preDraw(ExportTask exportTask, Graphics2D graphics, int width, int height, PlottingTask plottingTask){
-        graphics.translate(exportTask.exportResolution.getScaledOffsetX(), exportTask.exportResolution.getScaledOffsetY());
+        graphics.translate(exportTask.exportResolution.getScaledOffsetX()+0.5, exportTask.exportResolution.getScaledOffsetY()+0.5);
```

Comment:
In some situations like SVG / PDF exporting the _getScaledOffset_ will not be relative to the printing size not the image size, so 0.5 may not be half a pixel here. I think the 0.5 offset would need to added earlier in the chain, either in the PFM itself or in the "Print Resolution" transforms.

Reply:
My limited testing of PDF and SVG output showed this fix to be working adequately, but I may be missing something. Can you describe cases where the result would not be as expected? (Btw, it seems to me that addressing this issue in the PFM itself would be a little _too_ early in the chain ;)
And sorry - I probably should have put this on a separate branch... (pushed too soon!)